services/api-server/src/simcore_service_api_server/api/routes/studies_jobs.py

```diff
@@ -274,25 +273,56 @@ async def get_study_job_output_logfile(study_id: StudyID, job_id: JobID):
     "/{study_id}/jobs/{job_id}/metadata",
     response_model=JobMetadata,
     include_in_schema=API_SERVER_DEV_FEATURES_ENABLED,
-    status_code=status.HTTP_501_NOT_IMPLEMENTED,
 )
 async def get_study_job_custom_metadata(
```

Comment:
the default `status_code` in `fastapi` router decorators is 200 except indicated otherwise.
Here also `include_in_schema=False` so the OAS is not affected.